### ISSUE TITLE
HistoryTemplateEmailMappingsTests.testEmailFields failure in CI

### ISSUE THREAD

**Issue by DaveCTurner** [elastic/elasticsearch#65688]
**Build scan**: https://gradle-enterprise.elastic.co/s/r5l7htjefbfkm/console-log?task=:x-pack:plugin:watcher:internalClusterTest

**Repro line**: `./gradlew ':x-pack:plugin:watcher:internalClusterTest' --tests "org.elasticsearch.xpack.watcher.history.HistoryTemplateEmailMappingsTests.testEmailFields" -Dtests.seed=D2DECCA5D6646B13 -Dtests.security.manager=true -Dtests.locale=en-NZ -Dtests.timezone=America/Guadeloupe -Druntime.java=11`

**Reproduces locally?**: No

**Applicable branches**: Only seen on `master`.

**Failure history**: This test had a spate of failures recently, tracked in https://github.com/elastic/elasticsearch/issues/65052 and apparently resolved, but this does not look to be an instance of the same failure.

**Failure excerpt**:

```

2> java.lang.AssertionError: |  
-- | --
  | Expected: is <1L> |  
  | but: was <2L> |  
  | at __randomizedtesting.SeedInfo.seed([D2DECCA5D6646B13:A43F711F189A7FDD]:0) |  
  | at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18) |  
  | at org.junit.Assert.assertThat(Assert.java:956) |  
  | at org.junit.Assert.assertThat(Assert.java:923) |  
  | at org.elasticsearch.xpack.watcher.history.HistoryTemplateEmailMappingsTests.testEmailFields(HistoryTemplateEmailMappingsTests.java:100)

```

**Comment by dimitris-athanasiou** [comment]
Another failure during PR CI: https://gradle-enterprise.elastic.co/s/o7lngechrr5gq

**Comment by probakowski** [comment]
This looks like a duplicate of #67069 and it should be fixed by #67074.
Closing for now.
